packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -15,6 +15,13 @@ Before you set up the Apex Debugger, make sure that you have these essentials.
 * [Visual Studio Code](https://code.visualstudio.com/download) v1.17 or later
 * The latest versions of the [salesforcedx-vscode-core](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core) and [salesforcedx-vscode-apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex) extensions (we suggest that you install all extensions in the [salesforcedx-vscode](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) extension pack)
 
+---
+**NOTE:** Having an active installation of the Visual Studio Live Share extension stops the Apex Debugger from working. Before you use the Apex Debugger, set the `liveshare.excludedDebugTypes` setting to `["apex-replay"]`. For instructions, see [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) in the Visual Studio Code docs.
```

Comment:
We need to include the Apex Interactive Debugger as well. You would need to include https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex-debugger/package.json#L79

So, it should be `["apex-replay", "apex"]`

As an update, I'm still investigating the fix from Microsoft but haven't had a chance to do that yet. 

We can use this file and if I make progress we can just take this out from the release/v42.17.0 branch.